Catalogs/Vehicles/VehicleCatalog.xosc

```diff
@@ -3,76 +3,192 @@
   <FileHeader revMajor="1" revMinor="0" date="2020-07-10T10:00:00" author="BMW AG" description="Vehicle catalog" />
   <Catalog name="VehicleCatalog">
     <Vehicle name="car_ego" vehicleCategory="car">
+	  <Properties>
```

Comment:
Can we add a comment please that these Properties are necessary for openPASS?

Reply:
Added in c70afa2cbd24ca95bc2f198ba1be37ed9ebdb2a5